### PR TITLE
gui: Add translation fixes from localization team

### PR DESCRIPTION
**Pull request by reaganlo**
Add translation fixes from localization team

### CODE REVIEW

File: locale/es_MX/LC_MESSAGES/clr-installer.po [hunk 1]
```diff
@@ -321,7 +321,7 @@ msgid "Welcome to Clear Linux* OS Desktop Installation"
 msgstr "Bienvenido a la instalación del sistema operativo Clear Linux* para computadoras de escritorio"
 
 msgid "Thank you for choosing Clear Linux* OS"
-msgstr "Gracias por elegir el sistema operativo Clear Linux*"
+msgstr "Gracias por \nelegir el \nsistema \noperativo \nClear Linux*"
```
Comment (mdhorn) [inline locale/es_MX/LC_MESSAGES/clr-installer.po]: Why are they adding embedded newlines to the text? I thought we had worked hard to remove these?

File: locale/es_MX/LC_MESSAGES/clr-installer.po [hunk 2]
```diff
@@ -474,10 +474,10 @@ msgid "Passphrases do not match"
 msgstr "Las frases de contraseña no coinciden"
 
 msgid "Mapping %s partition to an encrypted partition"
-msgstr "Asignación de la partición %s a una partición cifrada"
+msgstr "Asignación de la partición% s a una partición cifrada"
```
Comment (mdhorn) [inline locale/es_MX/LC_MESSAGES/clr-installer.po]: I don't think this is valid -- they broke the `%s`

File: locale/es_MX/LC_MESSAGES/clr-installer.po [hunk 3]
```diff
@@ -474,10 +474,10 @@ msgid "Passphrases do not match"
...
 
 msgid "Skipping new file system for %s"
-msgstr "Omitir nuevo sistema de archivos para %s"
+msgstr "Omitir nuevo sistema de archivos para% s"
```
Comment (mdhorn) [inline locale/es_MX/LC_MESSAGES/clr-installer.po]: I don't think this is valid -- they broke the `%s`

File: locale/zh_CN/LC_MESSAGES/clr-installer.po [hunk 4]
```diff
@@ -471,13 +471,13 @@ msgid "Encryption Passphrase"
 msgstr "加密密码"
 
 msgid "Passphrases do not match"
-msgstr "密码短语不匹配"
+msgstr "密码不一致"
 
 msgid "Mapping %s partition to an encrypted partition"
-msgstr "将 %s 分区映射到加密分区"
+msgstr "将% s 分区映射到加密分区"
```
Comment (mdhorn) [inline locale/zh_CN/LC_MESSAGES/clr-installer.po]: I don't think this is valid -- they broke the `%s`

File: locale/zh_CN/LC_MESSAGES/clr-installer.po [hunk 5]
```diff
@@ -471,13 +471,13 @@ msgid "Encryption Passphrase"
...
 
 msgid "Skipping new file system for %s"
-msgstr "跳过 %s 的新文件系统"
+msgstr "跳过% s 的新文件系统"
```
Comment (mdhorn) [inline locale/zh_CN/LC_MESSAGES/clr-installer.po]: I don't think this is valid -- they broke the `%s`

File: locale/es_MX/LC_MESSAGES/clr-installer.po (hunk 1 above)
Comment (reaganlo) [reply locale/es_MX/LC_MESSAGES/clr-installer.po]: I noticed this. But It appeared fine on the page. So ignored. Maybe we can ask them.

File: locale/es_MX/LC_MESSAGES/clr-installer.po (hunk 2 above)
Comment (reaganlo) [reply locale/es_MX/LC_MESSAGES/clr-installer.po]: Good catch. Lets check with them.

**Pull request merged**